scripts/lib/api/processHtml.ts

```diff
@@ -287,8 +287,6 @@ export function processMembersAndSetMeta(
         const id = $dl.find("dt").attr("id") || "";
         const apiType = getApiType($dl);
 
-        const github = prepareGitHubLink($child, apiType === "method");
-
         if (child.name !== "dt" || !apiType) {
           return `<div>${$child.html()}</div>`;
         }
```

Comment:
Reversing the order of the `prepareGitHubLinks` call, and the if statement adds a bunch of new GitHub source links that we didn't have before. I'm not sure why yet, but the change seems desirable

Reply:
Ah. `prepareGitHubLink` calls `remove()` on the closest `<a>`. So it's removing the link for a child element when it shouldn't be. This would actually be good to have in a dedicated PR since it's a behavior change, and so that we can document it better in the PR description etc. No need to run `npm run regen-apis`. Instead, create a new branch based on this one, then interactive rebase to drop your first two commits and add a new one with the behavior change and no refactors.

Reply:
Ahh good spot! What do you think about making that change after merging this PR in a follow-up? By calling `prepareGitHubLink` inside of each child's `process` function, we end up with the same behavior as before plus we fixed some links

Reply:
That's fine with me to be in a follow up, but then you should revert the docs regen in this PR. It's important that `npm run regen-apis` doesn't result in changes when run on `main`, since that messes up other people making docs improvements.

Reply:
Done! This PR only refactors the code without changing the order of `prepareGitHubLinks`. In a follow-up, I'll add the change and the new links in the docs.